### DISCUSSION TITLE
Show issues, project's title and other things on profile!

### DISCUSSION THREAD

**Discussion by volkantash** [community/community#19827] [Other Feature Feedback, Questions, & Ideas]
Selam! / Hi!

Görevlendirildiğim issue, project ve diğer şeyleri niçin profilimde göremiyorum? 
> Why can't I see the issue, project and other things I've been assigned on my profile?

Şu an hiçbir projeyi izleyemiyorum. İzleyebilmem için proje içine girmeliyim. Proje başlığında üyeliğim bağlandığında haberim olmuyor. Olsa bile sonradan geçmişte yok oluyor. Bazılarını unutuyorum ve yarım kalmış işlere neden oluyor.
> I can't watch any projects right now. I have to get into the project to watch it. I do not know when my subscription is connected in the project title. Even if it does, it disappears later in the past. I forget some and it causes unfinished business.

**Comment by themiralay** [comment]
[This present issue](https://github.com/github-community/community/discussions/7575) is probably most popular issue with regard to tasks in Github Projects (Beta)!
 
 Other (possible) duplicates are:
 
 * [[Projects Beta] Show number of tasks vs completed task after issue title #16616](https://github.com/orgs/github-community/discussions/16616)
 * [[Projects Beta] Show x of n tasks on issue #13033](https://github.com/orgs/github-community/discussions/13033)
 * [[Projects beta] Task list progress #16721](https://github.com/orgs/github-community/discussions/16721)
 * [Show percent tasks complete in a new projects Beta field #12050](https://github.com/orgs/github-community/discussions/12050)
